### PR TITLE
feat: remove block header, add title

### PR DESCRIPTION
**Pull request by yuberdysheva**

### CODE REVIEW

File: src/components/Title/README.md [hunk 1]
```diff
@@ -1,12 +1,18 @@
 `type: "title"`
 
-`text: text` — Title text
+`title:`
```
Comment (aeksandla) [inline src/components/Title/README.md:3]: should there be a description of the prop here?

**Review by aeksandla — APPROVED** [review]

File: src/blocks/ExtendedFeatures/ExtendedFeatures.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ export const ExtendedFeaturesBlock = ({
 
     return (
         <AnimateBlock className={b()} animate={animated}>
-            <BlockHeader title={title} description={description} className={b('header')} />
+            <Title title={title} description={description} className={b('header')} />
```
Comment (gorgeousvlad) [inline src/blocks/ExtendedFeatures/ExtendedFeatures.tsx:33]: is it strange that component `Title` has property `title`?) Whole component is title
Or in this case `description` should became `subtitle` may be

File: src/components/Title/TitleItem.tsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+}
+
+export interface TitleItemFullProps extends TitleItemProps {
+    className?: string;
+    onClick?: () => void;
+    dataQa?: string;
+    resetMargin?: boolean;
+}
+
+const Title = (props: TitleItemFullProps) => {
```
Comment (gorgeousvlad) [inline src/components/Title/TitleItem.tsx:36]: `TitleItem`?

File: src/sub-blocks/Content/Content.tsx [hunk 4]
```diff
@@ -56,14 +56,14 @@ const Content = (props: ContentBlockProps & ClassNameProps) => {
...
-            : title;
+            ? ({text: title, textSize: getTextSize(size), resetMargin: true} as TitleItemProps)
+            : {...title, resetMargin: true};
 
     const hasTitle = Boolean(title);
 
     return (
         <Col className={b({size, centered, theme}, className)} reset sizes={colSizes}>
-            {title && <Title className={b('title')} {...titleProps} resetMargin />}
+            {title && <Title className={b('title')} title={{...titleProps}} colSizes={{all: 12}} />}
```
Comment (gorgeousvlad) [inline src/sub-blocks/Content/Content.tsx:66]: `title={titleProps}`

File: src/sub-blocks/Content/Content.tsx [hunk 5]
```diff
@@ -56,14 +56,14 @@ const Content = (props: ContentBlockProps & ClassNameProps) => {
 
     const {...titleProps} =
         !title || typeof title === 'string'
-            ? ({text: title, textSize: getTextSize(size)} as TitleProps)
-            : title;
+            ? ({text: title, textSize: getTextSize(size), resetMargin: true} as TitleItemProps)
+            : {...title, resetMargin: true};
```
Comment (gorgeousvlad) [inline src/sub-blocks/Content/Content.tsx:60]: `resetMargin` is true by default in `TitleItem`

**Review by gorgeousvlad — CHANGES_REQUESTED** [review]

File: src/blocks/ExtendedFeatures/ExtendedFeatures.tsx (hunk 2 above)
Comment (yuberdysheva) [reply src/blocks/ExtendedFeatures/ExtendedFeatures.tsx:33]: I renamed `description` to `subtitle`

**Review by gorgeousvlad — APPROVED** [review]

**Pull request merged**